centreon/tests/e2e/features/Additional-connectors/01-ACC-creation/index.ts

```diff
@@ -0,0 +1,204 @@
+});
+
+after(() => {
+  cy.stopContainers();
+});
+
+Given('a non-admin user is in the Specific Connector Configuration page', () => {
+  cy.loginByTypeOfUser({
+    jsonName: 'user-non-admin-for-ACC',
+    loginViaApi: false
+  });
+  cy.visit('/centreon/configuration/additional-connector-configurations');
+  cy.wait('@getConnectorPage');
+});
+
+When('the user clicks on Add', () => {
+  cy.getByLabel({ label: 'Add', tag: 'button' }).click();
+});
+
+Then('a pop-up menu with the form is displayed', () => {
+  cy.contains('Create additional connector configuration').should('be.visible');
+});
+
+When('the user fills in all the informations', () => {
+  cy.getByLabel({ label: 'Name', tag: 'input' }).type('Connector-001');
+  cy.getByLabel({ label: 'Description', tag: 'textarea' }).type("I'm the first connector created");
+  cy.get('#mui-component-select-type').should('have.text', 'VMWare 6/7');
+  cy.getByLabel({ label: 'Select poller(s)', tag: 'input' }).click();
+  cy.contains('Central').click();
+  cy.get('[id="Usernamevalue"]').type('admin');
+  cy.get('[id="Passwordvalue"]').type('Abcde!2021');
+  cy.get('[id="vCenternamevalue"]').clear().type('vCenter-001');
+  cy.get('[id="URLvalue"]').clear().type('https://10.0.0.0/sdk');
+  cy.get('[id="Portvalue"]').should('have.value', '5700');
+});
+
+When('the user clicks on Create', () => {
+  cy.getByLabel({ label: 'Create', tag: 'button' }).click();
+});
+
+Then('the first connector is displayed in the Specific Connector Configuration page', () => {
+  cy.wait('@addAdditionalConnector');
+  cy.get('*[role="rowgroup"]')
+    .should('contain', 'Connector-001');
+});
+
+When('the user fills in the mandatory informations', () => {
+  cy.getByLabel({ label: 'Name', tag: 'input' }).type('Connector-002');
+  cy.get('#mui-component-select-type').should('have.text', 'VMWare 6/7');
+  cy.getByLabel({ label: 'Select poller(s)', tag: 'input' }).click();
+  cy.contains('Poller-1').click();
+  cy.get('[id="Usernamevalue"]').type('admin');
+  cy.get('[id="Passwordvalue"]').type('Abcde!2021');
+  cy.get('[id="vCenternamevalue"]').clear().type('vCenter-002');
+  cy.get('[id="URLvalue"]').clear().type('https://10.0.0.0/sdk');
+  cy.get('[id="Portvalue"]').should('have.value', '5700');
+});
+
+Then('the second configuration is displayed in the Specific Connector Configuration page', () => {
+  cy.wait('@addAdditionalConnector');
+  cy.get('*[role="rowgroup"]')
+    .should('contain', 'Connector-002');
+});
+
+Then('the third configuration is displayed in the Specific Connector Configuration page', () => {
+  cy.wait('@addAdditionalConnector');
+  cy.get('*[role="rowgroup"]')
+    .should('contain', 'Connector-003');
+});
+
+When('the user clicks to add a second vCenter', () => {
+  cy.getByLabel({ label: 'Add vCenter/ESX', tag: 'button' }).click();
+});
+
+Then('a second group of parameters is displayed', () => {
+  cy.get('[class="css-1x7gjqg-parametersContainer"]').eq(0).should('exist');
```

Comment:
**The current selector:**
cy.get('[class="css-1x7gjqg-parametersContainer"]').eq(0).should('exist');

uses a dynamic class (css-1x7gjqg), which is likely to change over time. This can cause the test to fail unnecessarily whenever the class changes. To make the test more reliable, it would be better to avoid targeting dynamic parts of class names.

Instead, we can use a more stable selector that only targets elements whose class name ends with -parametersContainer, as shown below:

cy.get('[class$="-parametersContainer"]').eq(0).should('exist');
This will ensure that our tests remain stable even when the dynamic portion of the class changes.